src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -188,4 +190,38 @@
    :margin-top       2
    :margin-bottom    4
    :margin-right     2
-   :background-color :white})
\ No newline at end of file
+   :background-color :white})
+
+(def public-chats-item-container
+  {:flex-direction :row
+   :padding        3})
+
+(def public-chats-icon-container
+  {:flex 0.15})
```

Comment:
Magic number?

Reply:
Mostly all numbers in styles are magic :)